### PR TITLE
chore(TS): minor style

### PR DESCRIPTION
**Pull request by ShaMan123**
## Motivation

Improve some typings

### CODE REVIEW

File: src/shapes/Text/constants.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ const fontProperties = [
   'fontStyle',
 ] as const;
 
-const textDecorationProperties = [
+export const textDecorationProperties = [
```
Comment (asturur) [inline src/shapes/Text/constants.ts:11]: do we need to export this?

Comment (asturur) [reply src/shapes/Text/constants.ts:11]: I think maybe is a leftover of trying to get the type automatically from this array?

Comment (ShaMan123) [reply src/shapes/Text/constants.ts:11]: Something like that
I used it for the HTML logic

**Review by asturur — APPROVED** [review]

**Comment by asturur** [comment]
I added the changelog manually here because for some reason the action can't add it here.
If is all green i ll merge

File: src/shapes/Text/constants.ts (hunk 1 above)
Comment (ShaMan123) [reply src/shapes/Text/constants.ts:11]: Not really

**Pull request merged**